spec/presto/metrics_spec.rb

```diff
@@ -6,6 +6,6 @@
   end
 
   it 'does something useful' do
-    expect(false).to eq(true)
+    expect(true).to eq(true)
```

Comment:
@xerial I'm not fully sure what this test is. Just confirmation for failing CI. Right?

Reply:
I think it was a sanity test.